learning_journal/__init__.py

```diff
@@ -14,8 +14,11 @@ def main(global_config, **settings):
     DBSession.configure(bind=engine)
     Base.metadata.bind = engine
     config = Configurator(settings=settings)
-    config.include('pyramid_chameleon')
+    config.include('pyramid_jinja2')
     config.add_static_view('static', 'static', cache_max_age=3600)
-    config.add_route('home', '/')
+    config.add_route('list', '/')
+    config.add_route('detail', '/entry/{id}')
+    config.add_route('add_entry', '/add_entry')
+    config.add_route('edit_entry', '/entry/{id}/edit')
```

Comment:
Given the structure of your other two URLs, it might make more sense to have the pattern for the `add_entry` view be something like: `/entry/add`